src/SFA.DAS.Admin.Aan.Web/Controllers/CreateEvent/GuestSpeakerListController.cs

```diff
@@ -0,0 +1,50 @@
+﻿
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Admin.Aan.Application.Services;
+using SFA.DAS.Admin.Aan.Web.Infrastructure;
+using SFA.DAS.Admin.Aan.Web.Models.NetworkEvent;
+
+namespace SFA.DAS.Admin.Aan.Web.Controllers.CreateEvent;
+
+[Authorize]
+[Route("manage-events/new/guest-speaker-list", Name = RouteNames.CreateEvent.GuestSpeakerList)]
```

Comment:
The route seems odd to start with `manage-events`, can the routes be like
- `events` to look for events
- `events/{id}` to get to event details
- `events/new/guestspeakers` to get guest speakers list
- `events/new/guestspeakers/add` to add guest speaker
- `events/new/guestspeakers/question` to ask if they want guest speakers

Reply:
I will do this for all events pages done so far